pkg/reconciler/v1alpha1/broker/broker.go

```diff
@@ -151,7 +151,7 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		r.recorder.Event(broker, corev1.EventTypeNormal, brokerReconciled, "Broker reconciled")
```

Comment:
Please ignore broker.go. This file slipped in due to some other change I had in it